code/modules/power/engines/supermatter/supermatter.dm

```diff
@@ -799,6 +801,10 @@
 	playsound(get_turf(src), 'sound/effects/supermatter.ogg', 50, TRUE)
 	Consume(AM)
 
+/obj/machinery/atmospherics/supermatter_crystal/Bump(atom/A, yes)
```

Comment:
"yes"?

Reply:
it's on the parent call like this as well /shrug